lib/Dist/Zilla/Dist/Builder.pm

```diff
@@ -589,21 +588,23 @@ by the loaded plugins.
 sub release {
   my $self = shift;
 
-  Carp::croak("you can't release without any Releaser plugins")
-    unless my @releasers = @{ $self->plugins_with(-Releaser) };
+  $self->log_fatal("you can't release without any Releaser plugins")
```

Comment:
IMHO -- This should probably be pulled out into a separate change, because it doesn't really relate to this PR.